jobs/ci-run/integration/gen/test-user.yml

```diff
@@ -32,16 +32,128 @@
     - multijob:
         name: 'IntegrationTests-user'
         projects:
-        - name: 'test-user-lxd'
+        - name: 'test-user-test-user-login-password-lxd'
           current-parameters: true
-        - name: 'test-user-aws'
+        - name: 'test-user-test-user-login-password-aws'
```

Comment:
Do we really want the duplicate wording there? `test-user-test-user-login-password-aws`

Reply:
This code is auto-generated.
Template is: test-<suite-name>-test-<suite-name>-<test-name>-<cloud-name>

So it's not a part of this PR. But we can discuss changes in the template for code generation.